From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Marco Schirrmeister <mschirrmeister@gmail.com>
Date: Wed, 24 Jun 2026 05:00:00 +0000
Subject: arm: dts: rk3576-nanopi-r76s: add u-boot specific dtsi

Upstream U-Boot support for the NanoPi R76S is fresh, but it needs 
a few local adjustments to handle the boot process properly.

This patch adds the u-boot specific DTSI to:
- Include the base RK3576 u-boot definitions.
- Set the system heartbeat LED (led-0) to stay 'on' during the 
  bootloader phase as a visual 'power-on' indicator.

Signed-off-by: Marco Schirrmeister <mschirrmeister@gmail.com>
---

diff --git a/arch/arm/dts/rk3576-nanopi-r76s-u-boot.dtsi b/arch/arm/dts/rk3576-nanopi-r76s-u-boot.dtsi
new file mode 100644
index 00000000000..9798c46e2d0
--- /dev/null
+++ b/arch/arm/dts/rk3576-nanopi-r76s-u-boot.dtsi
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+
+#include "rk3576-u-boot.dtsi"
+
+&{/gpio-leds/led-0} {
+	default-state = "on";
+};
